main.py

```diff
@@ -3,8 +3,35 @@
 from CommandsAndAdaptors.parser_adaptor import *
 
 try:
-    input_array = ParsedInput(parse_args())
+    input_array = ParsedInput.parse(parse_args())
     command = CommandFactory(input_array).create()
     output_message = command.execute()
+    status = 0
+except NoCommandError:
+    status = 1
+except WrongCommandError:
+    status = 2

```

Comment:
The reason behind the enum that you don’t need to deal with integer and deal
With enum.

status = success not status = 0

Reply:
What are you going to print to the client here ?
You should have the following 
command_output = None
Command= factory.create()
Try{
Result = Command.execute()
commandOutput= CommandOutput(result)
} catch (Exception e ) {
command_output = commandOutput(exception)
}

print (toJson(command_output))

this will be more cleaner